### PR TITLE
OCM-152 | feat: 'rosa verify network' command

### PR DESCRIPTION
**Pull request by thomasmckay**
[OCM-152](https://issues.redhat.com//browse/OCM-152)

https://docs.google.com/document/d/1pLN83t5kh7Pv0iBtJFFQnIQvdpagG4WZIyel9YRFE3s/edit#

Note: The `go mod vendor` commit will be removed. Dependent on:
https://github.com/openshift-online/ocm-api-model/pull/757
https://gitlab.cee.redhat.com/service/uhc-clusters-service/-/merge_requests/5884

### CODE REVIEW

**Comment by thomasmckay** [comment]
@willkutler Removing `--status-only` would imply removing the `--watch` too which means it would always be "interactive". I haven't used rosa enough to know users' patterns. Would it be sufficient to just always block and wait when the command is run?

**Comment by willkutler** [comment]
@thomasmckay Hmm, thinking it about it more, it's probably better to not require users to be blocked and wait for a minute-plus long command to run. I think it should be good as-is

File: cmd/verify/network/cmd.go [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+		}
+
+		region, err := aws.GetRegion(arguments.GetRegion())
+		if err != nil {
+			r.Reporter.Errorf("Error getting region: %v", err)
+			os.Exit(1)
+		}
+		args.region = region
+	} else if cmd.Flags().Changed(clusterFlag) {
+		if cmd.Flags().Changed(subnetIDsFlag) {
```
Comment (gdbranco) [inline cmd/verify/network/cmd.go:124]: Couldn't this check and the one above be merged into a single condition which checks if both have changed?

Comment (gdbranco) [reply cmd/verify/network/cmd.go:124]: I don't mind that much, but personally find it cleaner

**Review by renan-campos — APPROVED** [review]
/lgtm

**Pull request merged**